### PR TITLE
Add engine merge method configuration

### PR DESCRIPTION
**Pull request by liyuqian**
In order for this to take effect, https://skia-review.googlesource.com/c/buildbot/+/167200
needs to be landed and deployed.

The `/bin/internal/engine.merge_method` file controls how we merge a pull
request created by the engine auto-roller. If it's `squash`, there's only one
commit for a pull request no matter how many engine commits there are inside
that pull request. If it's `rebase`, the number of commits in the framework is
equal to the number of engine commits in the pull request. The latter method is
easier to detect regressions but cost more test resources.

### CODE REVIEW

**Review by goderbauer — COMMENTED** [review]
Did you by any chance forget to include the `engine.merge_method` file? Or is it missing on purpose?

**Comment by liyuqian** [comment]
Ah, I forgot `engine.merge_method`. Thanks!

**Review by goderbauer — APPROVED** [review]
LGTM

File: bin/internal/README.md [hunk 1]
```diff
@@ -4,3 +4,10 @@ Dart SDK dependency
 The `bin/internal/engine.version` file controls which version of the Flutter engine to use.
 The file contains the commit hash of a commit in the <https://github.com/flutter/engine> repository.
 That hash must have successfully been compiled on <https://build.chromium.org/p/client.flutter/> and had its artifacts (the binaries that run on Android and iOS, the compiler, etc) successfully uploaded to Google Cloud Storage.
+
+The `/bin/internal/engine.merge_method` file controls how we merge a pull
+request created by the engine auto-roller. If it's `squash`, there's only one
+commit for a pull request no matter how many engine commits there are inside
+that pull request. If it's `rebase`, the number of commits in the framework is
+equal to the number of engine commits in the pull request. The latter method is
+easier to detect regressions but cost more test resources.
```
[the remark this answers is not in the document]
Comment (liyuqian) [reply bin/internal/README.md]: Done. Thanks!

**Review by dnfield — APPROVED** [review]

**Pull request merged**